src/components/EventDetails/EventDetails.js

```diff
@@ -931,7 +942,9 @@ class EventDetail extends React.Component<Props, State> {
             squarePrimary: true,
           };
 
-          if (isReceived) {
+          if (isReferralRewardTransaction) {
+            eventData.buttons = [];
```

Comment:
```suggestion
           buttons = [];
```

Reply:
> We have designs for referrals in activity feed and event detail modals
> https://zpl.io/2jnQ7gQ
> https://zpl.io/bLlPGv7
> but if it's outside of the scope of this PR it's fine as it is 👍

Aahh, wasn't aware of that ;/
Will add designs in other PR then :)